cypress/integration/pages/mediaAssetPage/tests.js

```diff
@@ -66,12 +66,36 @@ export const testsThatFollowSmokeTestConfig = ({
               .eq(1)
               .should(
                 'contain',
-                appConfig[config[service].name][variant].articleTimestampPrefix,
+                appConfig[service][variant].articleTimestampPrefix,
               );
           }
         },
       );
     });
+
+    it('should navigate to first related content link, and back again', () => {
+      cy.request(`${config[service].pageTypes[pageType].path}.json`).then(
+        ({ body }) => {
+          const arrayLength = body.relatedContent.groups.length;
+
+          if (arrayLength > 0) {
+            const assetURI =
+              body.relatedContent.groups[0].promos[0].locators.assetUri;
```

Comment:
This seems prone to throw exceptions, unless we know that `promos[0].locators.assetUri` is always defined as long as `groups` has a length?

Reply:
On all the example pages on local and test this seemed to be fine (always was defined if there was content in groups, if no content in groups it skips the test), but I can have a look at a few more pages to check this.